lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
Good point. The behavior in Classic is that all four issues have the same fingerprint (which actually, also seems a little problematic, depending how we using the fingerprinting). 

Maybe we could just nuke line 115?